app/assets/javascripts/darkswarm/all.js.coffee

```diff
@@ -1,4 +1,4 @@
-#= require jquery
+#= require jquery2
```

Comment:
Just to be sure to understand. Why not using yarn (and `package.json`) for this one?

Reply:
So this is currently provided by a gem (jquery-rails) and that gem contains different versions of the jquery code which is sightly customised. At some point we can look at moving all javascript assets to being imported and installed via yarn and webpacker, but this was the simplest and quickest way to switch to `2.x`.

Reply:
Thanks!